Changelog.md

```diff
@@ -76,6 +76,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - extensions/ext/ray_tracing_pipeline: Pass indirect SBT regions as single item reference (#829)
 - Replaced `c_char` array setters with `CStr` setters (#831)
 - `push_next()` functions now allow unsized `p_next` argument (#855)
+- Flattened `ash::extensions` into `ash`, flattened `ash::vk::*` extension modules into `ash::vk`, and moved `*Fn` function pointer table structs from `ash::vk` into `ash` or the associated extension module (#894)
```

Comment:
>  flattened `ash::vk::*` extension modules into `ash::vk`

That seems slightly strange to mention as extension modules didn't exist (not even in `ash::vk::*`) in the latest release (they've only been introduced and resided there temporarily).

However, looks like we totally forgot to even mention that entire `mod`-refactor for extensions in #734.

Alas, nothing too bad, as I think we should write a proper actionable summary when publishing the `0.38` release.